pkg/networkmanager/index.html

```diff
@@ -136,17 +136,21 @@
   </script>
 
   <script id="network-ethernet-settings-template" type="x-template/mustache">
-    <table class="form-table-ct">
-      <tr>
-        <td>
-          <label class="control-label" translatable="yes">MTU</label>
-        </td>
-        <td>
-          <input id="network-ethernet-settings-mtu-input" class="form-control" type="text"
-                 value="{{mtu}}"></input>
-        </td>
-      </tr>
-    </table>
+    <div>
+      <label>
+        <input id="network-ethernet-settings-mtu-auto" {{^mtu}}checked="checked"{{/mtu}}
+               type="radio" name="ethernet-mtu-mode"></input>
+        <span translatable="yes">Automatic</span>
+      </label>
+    </div>
+    <div>
+      <label>
+        <input id="network-ethernet-settings-mtu-custom"  {{#mtu}}checked="checked"{{/mtu}}
+               type="radio" name="ethernet-mtu-mode"></input>
+        <span translatable="yes">Set to</span>
+        <input id="network-ethernet-settings-mtu-input" type="text" value="{{mtu}}"></input>
```

Comment:
This needs a form-control class.
This unfortunately makes the input end up on it's own row, so need some override css for doing display: inline;